builtin/stash.c

```diff
@@ -1499,22 +1504,28 @@ static int do_create_stash(const struct pathspec *ps, struct strbuf *stash_msg_b
 	return ret;
 }
 
-static int create_stash(int argc, const char **argv, const char *prefix UNUSED,
+static int create_stash(int argc, const char **argv, const char *prefix,
 			struct repository *repo UNUSED)
 {
 	int ret;
+	int include_untracked = 0;
+	struct option options[] = {
+		OPT_BOOL('u', "include-untracked", &include_untracked,
+			N_("include untracked files")),
+		OPT_END()
+	};
+
 	struct strbuf stash_msg_buf = STRBUF_INIT;
 	struct stash_info info = STASH_INFO_INIT;
 	struct pathspec ps;
 
-	/* Starting with argv[1], since argv[0] is "create" */
-	strbuf_join_argv(&stash_msg_buf, argc - 1, ++argv, ' ');
+	argc = parse_options(argc, argv, prefix, options, git_stash_create_usage, 0);
+
+	strbuf_join_argv(&stash_msg_buf, argc, argv, ' ');
 
 	memset(&ps, 0, sizeof(ps));
-	if (!check_changes_tracked_files(&ps))
-		return 0;
```

Comment:
@SBhojani I am fairly certain that this is needed, and that this accidental removal is the underlying root cause for the CI build to fail.

This should fix it:

```diff
diff --git a/builtin/stash.c b/builtin/stash.c
index e9eb15b53407..34828d1e2209 100644
--- a/builtin/stash.c
+++ b/builtin/stash.c
@@ -1524,6 +1524,8 @@ static int create_stash(int argc, const char **argv, const char *prefix,
 	strbuf_join_argv(&stash_msg_buf, argc, argv, ' ');
 
 	memset(&ps, 0, sizeof(ps));
+	if (!check_changes_tracked_files(&ps))
+		return 0;
 
 	ret = do_create_stash(&ps, &stash_msg_buf, include_untracked, 0, 0, &info,
 			      NULL, 0);
```